src/types/models/tracking.ts

```diff
@@ -0,0 +1,8 @@
+import { Language } from "../../types/params"
+
+export type WhatsappEntry = Record<
+  "firstName" | "language" | "lastName" | "phone",
+  string
+>
```

Comment:
what's the reason to reach for a type and `Record` instead of using an interface as we usually do?

Reply:
It's related to the dynamic type https://github.com/carforyou/carforyou-api-client-pkg/pull/225/files#diff-6d79f7439fbf4ceeb2d6e0ff0a243f47a09b9caf5acc63d081209fdac9b8ebd4R5 as you should not type it as `object` (see: [banned types](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/ban-types.md)) maybe I misinterpreted something here but II understood it as the recommended way with TS 2.0, the resulting type will be the same in this case but it matches the dynamic type passed to the extracted function